### PR TITLE
[Feat] Supports Aclgraph for Bert

### PR DESCRIPTION
**Pull request by JC-ut0**
### What this PR does / why we need it?

[Feat] Supports Aclgraph for Bert

### CODE REVIEW

File: vllm_ascend/patch/worker/patch_common/patch_roberta.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                                                    TOKEN_TYPE_MULTIPLIER)
+
+
+def _decode_token_type_ids(input_ids: torch.Tensor) -> torch.Tensor:
+
+    token_type_ids = input_ids // TOKEN_TYPE_MULTIPLIER
+
+    input_ids.bitwise_and_(TOKEN_MASK)
+
+    return token_type_ids
```
Comment (wangxiyuan) [inline vllm_ascend/patch/worker/patch_common/patch_roberta.py:46]: why not just patch these two func instead of model change?

**Review by wangxiyuan — COMMENTED** [review]
need rebase as well

File: vllm_ascend/patch/worker/patch_common/patch_roberta.py (hunk 1 above)
Comment (JC-ut0) [reply vllm_ascend/patch/worker/patch_common/patch_roberta.py:46]: These two functions are not sub-methods of any class, and they will be directly imported from bert.py in roberta.py

File: vllm_ascend/patch/worker/patch_common/__init__.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
 import vllm_ascend.patch.worker.patch_common.patch_attention_layer  # noqa
 import vllm_ascend.patch.worker.patch_common.patch_distributed  # noqa
 import vllm_ascend.patch.worker.patch_common.patch_logits  # noqa
+import vllm_ascend.patch.worker.patch_common.patch_roberta  # noqa
```
Comment (Yikun) [inline vllm_ascend/patch/worker/patch_common/__init__.py:28]: I haven't see any remove plan on https://github.com/vllm-project/vllm-ascend/blob/main/vllm_ascend/patch/__init__.py

Please add it.

Comment (JC-ut0) [reply vllm_ascend/patch/worker/patch_common/__init__.py:28]: Added

File: vllm_ascend/attention/attention_mask.py [hunk 3]
```diff
@@ -75,6 +76,14 @@ def get_attn_mask(self, max_seq_len: int, dtype: torch.dtype,
         return self.attn_mask_cache[:max_seq_len, :max_seq_len].contiguous(
         ).to(device, non_blocking=True)
 
+    def get_pooling_mask(self, device):
+        if self.pooling_mask is None:
+            # the compressed attention mask for npu_fusion_attention sparse mode 4
+            self.pooling_mask = torch.triu(torch.ones(
```
Comment (MengqingCao) [inline vllm_ascend/attention/attention_mask.py:82]: Does the attention operator needs a upper triangular matrix instead of an all-one matrix?

File: vllm_ascend/worker/model_runner_v1.py [hunk 4]
```diff
@@ -2624,7 +2644,6 @@ def initialize_kv_cache(self, kv_cache_config: KVCacheConfig) -> None:
         """
         kv_cache_config = deepcopy(kv_cache_config)
         self.kv_cache_config = kv_cache_config
-        self.initialize_attn_backend(kv_cache_config)
```
Comment (MengqingCao) [inline vllm_ascend/worker/model_runner_v1.py:2627]: `initialize_attn_backend` should done before `may_reinitialize_input_batch` to get the correct block size in hybrid kvcache scenario

File: vllm_ascend/attention/attention_mask.py (hunk 3 above)
Comment (JC-ut0) [reply vllm_ascend/attention/attention_mask.py:82]: This is the compressed attention mask for npu_fusion_attention sparse mode 4, the real mask will be initialized while calculating, and is controlled by `next_tockens`/ `pre_tockens`.
For more information, please refer to
https://www.hiascend.com/document/detail/zh/Pytorch/710/apiref/torchnpuCustomsapi/context/torch_npu-npu_fusion_attention.md
